packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -112,6 +118,9 @@ const initialState: SuiteState = {
         showDashboardT2B1PromoBanner: true,
         showSettingsDesktopAppPromoBanner: true,
     },
+    evmSettings: {
+        confirmExplanationModalClosed: {},
```

Comment:
It seems `flags` is not the right place because it's `[key]: boolean` only object (action `SUITE.SET_FLAG` requires boolean), my case is actually `confirmExplanationModalClosed: {[network]: boolean}` object so I have dynamic key based on network

Reply:
oka, makes sense